SayTheirNames/Source/Dependency/DependencyContainer.swift

```diff
@@ -24,23 +24,63 @@
 
 import UIKit
 
-protocol Servicing {
-    var image: ImageService { get }
-    var dateFormatter: DateFormatterService { get }
-    var network: NetworkRequestor { get }
+// MARK: - Dependency
+protocol Dependency {
+    
 }
-/// This is a core class that holds all instances responsible for logic
-final class Service: Servicing {
+
+// MARK: - DependencyContainer
+final class DependencyContainer: Dependency {
+    lazy private(set) var navigator = Navigator()
     lazy private(set) var image = ImageService()
     lazy private(set) var dateFormatter = DateFormatterService()
     lazy private(set) var network = NetworkRequestor()
     
-    static let shared = Service()
-    
     // MARK: - Init
     init() {
         Log.mode = .all
         Log.print("SayTheirNames Version: \(Bundle.versionBuildString)")
         Log.print("Starting Services")
+        
+        // Handle our Injection
+        let injectionFactory = DependencyInjectionFactory.notShared
```

Comment:
`notShared`? 🤔 😅  a bit confused, could you please explain?

Reply:
I mean it is shared but not realllly shared. Haha. Ill change it back.